From: Hilko Bengen <bengen@debian.org>
Date: Mon, 25 Aug 2014 18:52:58 +0200
Subject: Install vim addons into correct directory

---
 doc/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index b43f97a..0d0eab3 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -3,7 +3,7 @@ SUBDIRS = naturaldocs
 
 EXTRA_DIST = lenses.tex unambig.tex syntax/augeas.vim ftdetect/augeas.vim bcprules.sty xpath.txt
 
-vimdir = $(datadir)/vim/vimfiles
+vimdir = $(datadir)/vim/addons
 nobase_vim_DATA = syntax/augeas.vim ftdetect/augeas.vim
 
 # PDF targets
